converters/provider-aws/cloudfront/distribution.go

```diff
@@ -50,13 +50,34 @@ func DistributionResource(mg resource.Managed) ([]resource.Managed, error) {
 	target.Spec.ForProvider.ViewerCertificate[0].MinimumProtocolVersion = source.Spec.ForProvider.DistributionConfig.ViewerCertificate.MinimumProtocolVersion
 
 	target.Spec.ForProvider.WebACLID = source.Spec.ForProvider.DistributionConfig.WebACLID
+	if source.Spec.ForProvider.DistributionConfig.Origins.Items != nil {
```

Comment:
If the length of the `Items` slice is 0, then this for loop will be skipped. So, I am not sure that we really need this nil-check. However, the `Origins` is a pointer in the API definition and can be nil. In this case, we must add a nil check for `Origins` here.

Reply:
I'll fix